packages/module/src/AssistantMessageEntry/AssistantMessageEntry.tsx

```diff
@@ -14,19 +14,59 @@ const useStyles = createUseStyles({
     padding: "var(--pf-v5-global--spacer--sm) var(--pf-v5-global--spacer--md) var(--pf-v5-global--spacer--sm) var(--pf-v5-global--spacer--md)",
     maxWidth: "100%",
     wordWrap: "break-word",
+  },
+  label: {
+    backgroundColor: "var(--pf-v5-global--BackgroundColor--100)",
+    "--pf-v5-c-label__content--before--BorderColor": "#971d13",
+    "--pf-v5-c-label--PaddingBottom": ".3rem",
+    "--pf-v5-c-label--PaddingRight": "1rem",
+    "--pf-v5-c-label--PaddingLeft": "1rem",
+    "--pf-v5-c-label--PaddingTop": ".3rem",
+  },
+  activeOption: {
+    background: "var(--pf-v5-global--danger-color--100)",
+    pointerEvents: "none",
+    "--pf-v5-c-label__content--before--BorderColor": "#971d13",
+    "--pf-v5-c-label--m-outline__content--link--hover--before--BorderColor": "#971d13",
+    "--pf-v5-c-label__content--link--focus--before--BorderColor": "#971d13",
+    "& .pf-v5-c-label__content": {
+      color: "var(--pf-v5-global--BackgroundColor--100)",
+    },
+  },
+  disabledOption: {
+    background: "var(--pf-v5-c-label--m-red--BackgroundColor)",
+    opacity: "0.6",
+    pointerEvents: "none",
+    "--pf-v5-c-label__content--before--BorderColor": "var(--pf-v5-c-label--m-red--BackgroundColor) !important",
+    "& .pf-v5-c-label__content": {
+      color: "var(--pf-v5-c-label--m-red__content--Color)",
+    },
   }
 })
 
 interface AssistantMessageEntryProps {
   options?: {
     title: React.ReactNode;
-    props?: LabelProps
-  }[],
+    props?: LabelProps & { onClick?: () => void };
```

Comment:
`LabelProps` already has an onClick attribute, lets use that one.
We would need to pass down the event that we get.